src/shared/components/error-screen/ErrorScreen.scss

```diff
@@ -91,21 +92,20 @@
       margin-left: 25px;
     }
   }
-  
-  .vennIntersection{
+
+  .vennIntersection {
     padding-left: 19px;
-    .infoText{
+    .infoText {
       font-size: 18px;
       font-weight: bold;
     }
-    
+
     .infoIcon {
       svg {
         fill: $red;
-        width: 24px;
-        height: 24px;
+        --alert-icon-width: 20px;
+        --alert-icon-height: 20px;
```

Comment:
Make the AlertButton accept the `className` property that gets assigned to the outmost component. Then you'll be able to do `<AlertButton className={styles.alertIcon}>`, and you won't need the `.infoIcon` wrapper.

Reply:
Done